src/PrestaShopBundle/Controller/Admin/Sell/Customer/CustomerController.php

```diff
@@ -435,7 +435,14 @@ public function setRequiredFieldsAction(Request $request)
     public function searchAction(Request $request)
     {
         $query = $request->query->get('customer_search');
-        $phrases = explode(' ', $query);
+        $configuration = $this->get('prestashop.adapter.legacy.configuration');
+
+        if (!$configuration->get('PS_CUSTOMER_SEARCH_FULL')) {
+            $phrases = explode(' ', $query);
+        } else {
+            $phrases = [$query];
+        }
```

Comment:
```suggestion
        $phrases = $configuration->get('PS_CUSTOMER_SEARCH_FULL') ? [$query] : explode(' ', $query);
```

A small clarification of code. After that, it's an approval.

Reply:
Feedback applied